src/Graphics/src/Graphics/Size.cs

```diff
@@ -10,93 +10,62 @@ namespace Microsoft.Maui.Graphics
 	[TypeConverter(typeof(Converters.SizeTypeConverter))]
 	public partial struct Size
 	{
-		double _width;
-		double _height;
-
 		public static readonly Size Zero;
 
 		public Size(double size = 0)
 		{
-			if (double.IsNaN(size))
-				throw new ArgumentException("NaN is not a valid value for size");
-			_width = size;
-			_height = size;
+			Width = size;
```

Comment:
Could we include some unit tests accepting NaN values to cover any unexpected change/behavior in the future?